src/css/variables/_transitions.scss

```diff
@@ -0,0 +1,3 @@
+$enable-transitions: true;
```

Comment:
The logic shouldn't be switched on/off via SCSS variable. CSS should be stateless set of tools to use in theme/components. The decision should always be made on theme level so people have freedom to use any transition they want.

Reply:
Understood, variable removed and mixin edited...